bin/fix-make-bash

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
I faced some errors when cross compiled for the first time. Did not remember the error message. This script fix few things to enable cross compiling.